.devcontainer/devcontainer.json

```diff
@@ -17,29 +18,33 @@
 
 	// The optional 'workspaceFolder' property is the path VS Code should open by default when
 	// connected. This is typically a file mount in .devcontainer/docker-compose.yml
-	"workspaceFolder": "/usr/src/geonode/",
+	"workspaceFolder": "/usr/src",
```

Comment:
I would prefer to be dropped into the `geonode` folder. This would give me the chance to fire up `python manage.py xyz` commands once I am dropped into the container.